server.js

```diff
@@ -27,6 +30,19 @@ app.get('/test', (request, response) => {
 })
 
 app.get('/books', handleGetBooks);
+app.post('/books', handlePostBooks);
+app.delete('/books/:id', handleDeleteBooks);
+
+async function handleDeleteBooks (request, response) {
+  let id = request.params.id;
+  console.log(request.params);
+  try { 
+    await Books.findByIdAndDelete (id);
```

Comment:
- Ideally we can fix spacing for all methods: await Books.findByIdAndDelete  (id)  ->  
 `Books.findByIdAndDelete(id)` to follow conventional coding guidelines.

Also, in Canvas it was mentioned that you couldn't figure out why id isn't working. We are working with the id on the frontend and the backend.

What specific errors are you seeing?